### ISSUE TITLE
5.0.0rc4: Active link in table of contents not correct when nested content is expanded

### ISSUE THREAD

**Issue by renoger** [squidfunk/mkdocs-material#1548]
__I checked that...__

- [x] ... the documentation does not mention anything about my problem
- [x] ... the problem doesn't occur with the default MkDocs template
- [x] ... the problem is not in any of my customizations (CSS, JS, template)
- [x] ... there are no open or closed issues that are related to my problem

## Description

What I´m seeing now is that the active nav link (blue) on the right is jumping when I´m opening admonitions. Also when I have both admonitions open the last item in my nav is active - even though I can´t even scroll down that far.

![scroll_issue3](https://user-images.githubusercontent.com/62448529/77833319-c5479280-713c-11ea-9e97-c7692f46b43c.gif)

### Expected behavior

active link matches current page location

### Actual behavior

active link jumps when opening admonitions and doesn´t match page location

### Steps to reproduce the bug

1) Use a standard configuration
2) create a new .md site with below code and replace the path to a generic pdf file and see if you find the same behaviour

```
##a
---
##b

---

##c

###c1

??? info "test pdf"	
	<embed src="img/1.pdf" type="application/pdf" width="100%" height="480px">	
	
??? info "test pdf"	
	<embed src="img/1.pdf" type="application/pdf" width="100%" height="480px">

---

##d

###d1

###d2

###d3
```

### Package versions

* Python: 3.7.0
* MkDocs: 1.1
* Material: 5.0.0rc4

### System information

* OS: Win10
* Browser: Chrome

**Comment by squidfunk** [comment]
Thanks for reporting. The offsets of the headlines (i.e. permalink anchors) are currently not recalculated when the size of the main area changes.

**Comment by squidfunk** [comment]
Should be fixed with 7b6ce2a9.

**Comment by squidfunk** [comment]
Material for MkDocs 5 was just released, which includes this fix!
